core/eval.go

```diff
@@ -844,93 +897,101 @@ func evalQWATCH(args []string, c *Client) []byte {
 	return RESP_OK
 }
 
+// evalLIST returns an array of commands supported by DiceDB
+func evalLIST() []byte {
+	sort.Strings(diceCommands)
+	return Encode(strings.Join(diceCommands, ", "), false)
+}
+
 func executeCommand(cmd *RedisCmd, c *Client) []byte {
```

Comment:
Everything below this in this file is mostly cleanup to use the constant command names.